### PR TITLE
mermaid add and xmex transfer procedure for community project

### PR DESCRIPTION
**Pull request by ovidiuolteanu**

### CODE REVIEW

File: docs/developers/xmex-transfer.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+---
+sidebar_position: 3
+id: xmex-transfer
+---
+
+[comment]: # (mx-exclude-context)
+
+# XMEX transferability in community projects
+
+Projects developing on top of xExchange smart contracts might need the ability to work with XMEX, either acting as proxy to transfer it from/to the xExchange yield generating contracts, or by accepting it as means of payment for specific business cases. Many of the core principles of XMEX are designed around its account binding property, which in this case removed the open transferability of XMEX. For this reason, we have designed an XMEX transfer mechanisms for projects to use, in order to assure and streamline the conservation of the XMEX benefits and properties without too much development effort required on implementing project's side.
```
Comment (claudiulataretu) [inline docs/developers/xmex-transfer.md:10]: without too much development effort required from project's side.

File: docs/developers/xmex-transfer.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+[comment]: # (mx-context-auto)
+
+## Transfer method and specifications
+
+On each xExchange environment we have deployed an [XMEX wrapping contract](https://github.com/multiversx/mx-exchange-sc/tree/main/locked-asset/locked-token-wrapper) to facilitate the transferability for implementing projects. The XMEX wrapping contract accepts XMEX, removes its energy and properties from the sender and gives back WXMEX. The WXMEX token does not hold energy or benefits in its wrapped form, apart from it's ability to be transfered if the holder address is whitelisted.
```
Comment (claudiulataretu) [inline docs/developers/xmex-transfer.md:16]: to facilitate the transferability for projects building on top of xMEX

File: docs/developers/xmex-transfer.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+[comment]: # (mx-context-auto)
+
+## Transfer method and specifications
+
+On each xExchange environment we have deployed an [XMEX wrapping contract](https://github.com/multiversx/mx-exchange-sc/tree/main/locked-asset/locked-token-wrapper) to facilitate the transferability for implementing projects. The XMEX wrapping contract accepts XMEX, removes its energy and properties from the sender and gives back WXMEX. The WXMEX token does not hold energy or benefits in its wrapped form, apart from it's ability to be transfered if the holder address is whitelisted. 
+Therefore, the implementing projects will receive transferRole on the WXMEX tokenID on needed addresses, if their business criteria is accepted as in sync with the principles of XMEX.
+
+The whitelisted projects will need to wrap the XMEX they want to deposit or use. By wrapping it, the XMEX will temporarily “lose” its energy but the energy can be recovered and regained by users unWrap.
```
Comment (claudiulataretu) [inline docs/developers/xmex-transfer.md:19]: regained by users through unWrapping it back to xMEX

File: docs/developers/xmex-transfer.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+    end
+```
+
+[comment]: # (mx-context-auto)
+
+## Whitelisting process and criteria
+
+Implementing projects have to make a pull request in [the mx-specs repository](https://github.com/multiversx/mx-specs) containing a new file in the `LKMEX-whitelist` folder. The new file should contain the filled data required by the template you can find in [the last chapter of this page](#template-for-pr-contents). The name of the document should be the name of the project.
+
+Projects can also lose their transferRole status if the Smart Contract is being used in ways that go against LKMEX principles. Questionable activities will be reviewed by the MultiversX team and any Project or Smart Contract found in breach of the rules will automatically lose their transferRole status.
```
Comment (claudiulataretu) [inline docs/developers/xmex-transfer.md:93]: XMEX principles

File: docs/developers/xmex-transfer.md (hunk 3 above)
Comment (ovidiuolteanu) [reply docs/developers/xmex-transfer.md:19]: nice one. done.

File: docs/developers/xmex-transfer.md (hunk 2 above)
Comment (ovidiuolteanu) [reply docs/developers/xmex-transfer.md:16]: done.

File: docs/developers/xmex-transfer.md (hunk 4 above)
Comment (ovidiuolteanu) [reply docs/developers/xmex-transfer.md:93]: done.

**Review by claudiulataretu — APPROVED** [review]

**Pull request merged**